parquet-thrift/src/main/java/org/apache/parquet/thrift/ThriftParquetReader.java

```diff
@@ -1,14 +1,14 @@
-/* 
```

Comment:
This file has only style changes, would you mind reverting it?

Reply:
Valid point, done!

Reply:
I've reverted a couple more unrelated changes.